FirelandsProject/cataclysm
# Link Event 5 not found or invalid, skipped

### Changes proposed

- Correct the error that is presented inside the smart_scripts, since it has the wrong ids (triggered). Also, it has a link to an id that does not exist and it is not event 61, so it should not have a link.

### Issues addressed


- Closes

### SOURCE:


- SmartScript::ProcessAction: Entry 35456 SourceType 0, Event 4, Link Event 5 not found or invalid, skipped.

### Tests performed


- Windows 10
- In-game

### How to Test the Changes:


1. Enter the worgen zone.
2. When you try to go to the second part, you will notice this message on the console.

### Known issues and TODO list



---

- Thank you for collaborating with the project.
- For an efficient working methodology.
- It is obvious to make long pull request, unless it is necessary.
- A short pull request would be easy to test and approve.
